### ISSUE TITLE
Load Balancing

### ISSUE THREAD

**Issue by turowicz** [unifi-utilities/unifios-utilities#195]
I need to set up a virtual IP that will be load balancing across multiple clients in a UDM-PRO managed VLAN. Is it possible to achieve this with this project?

cc @boostchicken

**Comment by SturdyErde** [comment]
I'm sure you could run a containerized load balancer / reverse proxy such as NGINX, Traefik, or haproxy on UDM after setting up the base UDM Utilities.

**Comment by boostchicken** [comment]
@turowicz yeah its possible, care to get a bit more techincal on what you want to achieve

**Comment by turowicz** [comment]
I need to be able to host a kubernetes cluster behind a single IP in such a way, that all nodes are assigned traffic in a round-robin fashion.

**Comment by Kashalls** [comment]
I was looking at implementing ha-proxy using the host's networking on my UDMP. I got the container up and running, just having issues with networking on it using its own ip or the hosts.

**Comment by turowicz** [comment]
@Kashalls that would be awesome

**Comment by Kashalls** [comment]
> @Kashalls that would be awesome

I personally prefer using haproxy with the container networking, but some users might find it better to use a seperate ip. What do you think?

**Comment by turowicz** [comment]
I don't mind as long as its easy to set up

**Comment by Kashalls** [comment]
> I don't mind as long as its easy to set up

It's quite easy. Let me fork and pr.

**Comment by Kashalls** [comment]
> I don't mind as long as its easy to set up

https://github.com/boostchicken/udm-utilities/pull/239
